charts/redis-ha/templates/redis-ha-secret-init-ssh.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "redis-ha.fullname" . }}-init-ssh-env-secret
+  namespace: {{ .Release.Namespace }}
+  labels:
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app: {{ template "redis-ha.fullname" . }}
+type: Opaque
+data:
+  SSH_KEY: "{{ .Values.restore.ssh.key | b64enc }}"
```

Comment:
Why don't we just conditionally write the AWS/SSH keys and such in one secret file with `{{- if .Values.restore.xxx ]}`?